creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -683,83 +688,79 @@ class SnapbackSM {
    * Given map(replica set node => userWallets[]), retrieves clock values for every (node, userWallet) pair
    * @param {Object} replicaSetNodesToUserWalletsMap map of <replica set node : wallets>
    * @param {Set<string>} unhealthyPeers set of unhealthy peer endpoints
-   * @param {number?} [maxUserClockFetchAttempts=10] max number of attempts to fetch clock values
    *
-   * @returns {Object} map of peer endpoints to (map of user wallet strings to clock value of replica set node for user)
+   * @returns {Object} map(replica => map(wallet => clockValue))
```

Comment:
just to clarify replica is the single content node, not the replica set? if so, would you mind making that explicit here

Reply:
done